src/common/config.cpp

```diff
@@ -102,6 +108,7 @@ void load(const std::filesystem::path& path) {
             auto general = generalResult.unwrap();
 
             isNeo = toml::find_or<toml::boolean>(general, "isPS4Pro", false);
+            isFullscreen = toml::find_or<toml::boolean>(general, "Fullscreen", true);
```

Comment:
This setting should ideally have the same capitalization as other settings (first letter non-capital)